packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx

```diff
@@ -88,9 +100,25 @@ export const CanvasForm: FunctionComponent<CanvasFormProps> = (props) => {
       <Title headingLevel="h1">{componentName}</Title>
       <div className="canvas-form">
         {isUnknownComponent ? (
-          <CodeBlock>
-            <CodeBlockCode>{stringify(model)}</CodeBlockCode>
-          </CodeBlock>
+          <Card>
+            <CardHeader>
+              <CardTitle>Node source</CardTitle>
+            </CardHeader>
+            <CardBody>
+              <CodeBlock>
+                <CodeBlockCode>{stringify(model)}</CodeBlockCode>
+              </CodeBlock>
+            </CardBody>
+            <CardFooter>
+              <Hint>
+                <HintTitle>Node is not a known type</HintTitle>
+                <HintBody>
+                  Configuration properties are not available when the node is not a known type. Delete, or replace this
+                  node with valid node.
+                </HintBody>
+              </Hint>
+            </CardFooter>
+          </Card>
```

Comment:
I think this component is starting to grow too big to have here :), maybe it's time to move it to a dedicated file.

Reply:
sure - moved to separate file